docs/requirements-specification.md

```diff
@@ -21,4 +21,42 @@ The use cases fulfill the goals in the following way:
 | UC5 | G4, G8 |
 | UC6 | |
 
-The goals G10-G12 are currently not addressed by any use case.
\ No newline at end of file
+The goals G10-G12 are currently not addressed by any use case.
+
+### UC3: Get capability overview
+| UC3 | Get capability overview
```

Comment:
Since you specify multiple sub-use cases, you should introduce a unique index and name for each one (e.g., `UC3.1`, ...)